ProcessMaker/processmaker
# Make installing bundles a background/async job with status

## How to Test
Install a remote bundle in devlink. It's now a background job and you should see this progress window:

![image](https://github.com/user-attachments/assets/94806508-d3a6-48ef-bf15-c81edf2b7f28)


## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19520
-

## Code Review Checklist
